embassy-nrf/src/lib.rs

```diff
@@ -67,6 +67,8 @@ mod chip;
 #[path = "chips/nrf52840.rs"]
 mod chip;
 
+pub use chip::EASY_DMA_SIZE;
```

Comment:
I tried putting the docstring here, but it didn't get picked up. I can remove it if you prefer.